content/en/docs/concepts/traffic-management/index.md

```diff
@@ -13,237 +13,156 @@ aliases:
+external dependency of your mesh to the service registry. You can do all this
+and more by adding your own traffic configuration to Istio using Istio’s traffic
+management API.
+
+Like other Istio configuration, the API is implemented as Kubernetes custom
+resources (CRDs), which you can configure using YAML, as you’ll see in the
+examples.
+
+The rest of this guide looks at each of the traffic management API’s resources
+and what you can do with them. These resources are:
+
+-   [Virtual services](#virtual-services)
+-   [Destination rules](#destination-rules)
+-   [Gateways](#gateways)
+-   [Service entries](#service-entries)
+-   [Sidecars](#sidecars)
+
+This guide also gives an overview of some of the
+[network resilience and testing features](#network-resilience-and-testing) that
+are built in to the API’s resources.
+
+## Virtual services {#virtual-services}
+
+[Virtual services](/docs/reference/config/networking/v1alpha3/virtual-service/),
+along with destination rules, are the key building blocks of Istio’s traffic
+routing functionality. A virtual service lets you configure how requests are
+routed to a service within an Istio service mesh, building on the basic
+connectivity and discovery provided by Istio and your platform. Each virtual
+service consists of a set of routing rules that are evaluated in order, letting
+Istio match each given request to the virtual service to a specific real
+destination within the mesh. Your mesh can require multiple virtual services or
+none depending on your use case.
+
+### Why use virtual services? {#why-use-virtual-services}
+
+Virtual services play a key role in making Istio’s traffic management flexible
+and powerful. They do this by strongly decoupling where the client sends their
+requests from the destination workloads that actually implement them. Virtual
+services also provide a rich way of specifying different traffic routing rules
+for sending traffic to those workloads.
+
+So why is this so useful? Without virtual services, Envoy just distributes
+traffic using round-robin load balancing between all service instances, as
+described in the introduction. You can improve this behavior with what you know
+about the workloads: for example, some might represent a different version. This
+can be useful in A/B testing where you might want to configure traffic routes
+based on percentages across different service versions, or you might want to use
+a different load balancing policy for a particular set of instances.
+
+With virtual services, you create a new hostname which directs traffic to
+services of your choice, or overrides traffic behavior for an existing hostname.
+You can then use routing rules in the virtual service that tell Envoy how to
+send the virtual host’s traffic to the appropriate "real" workloads.
+
+A typical use case is to send traffic to different versions of a service,
+specified as service subsets. Clients send requests to the virtual service as if
```

Comment:
Do we want to say the send requests "to the virtual service"? Or they send requests to the service and the virtual service routes it?

Reply:
Maybe `Clients send requests to the virtual service host as if ...`?

Reply:
That works. Done.